bots/tools/beep.py

```
@@ -47,6 +47,45 @@
+
+    Args:
+        note_str: Note in scientific pitch notation (e.g., 'C4')
+
+    Returns:
+        MIDI note number (0-127)
+    """
+    note_str = note_str.strip()
+
+    # Extract note name and octave
+    if len(note_str) < 2:
+        raise ValueError(f"Invalid note format: {note_str}")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)